xc7/make/vivado.cmake

```diff
@@ -82,15 +121,18 @@ function(COMMON_VIVADO_TARGETS NAME WORK_DIR DEPS BITSTREAM)
   add_custom_target(${NAME} DEPENDS ${WORK_DIR}/design_${NAME}.dcp)
   add_custom_target(${NAME}_timing DEPENDS ${WORK_DIR}/timing_${NAME}.json)
   add_custom_target(${NAME}_fasm DEPENDS ${WORK_DIR}/design_${NAME}.bit.fasm)
-  add_custom_target(${NAME}_diff_fasm
-      COMMAND diff -u
-        ${BITSTREAM_LOCATION}.fasm
-        ${CMAKE_CURRENT_BINARY_DIR}/${WORK_DIR}/design_${NAME}.bit.fasm
-      DEPENDS
-        ${DEPS}
-        ${BITSTREAM_LOCATION}.fasm
-        ${CMAKE_CURRENT_BINARY_DIR}/${WORK_DIR}/design_${NAME}.bit.fasm
-      )
+
+  if(${COMMON_VIVADO_TARGETS_MAKE_DIFF_FASM})
```

Comment:
Do you need to canonicalize the file first?

Reply:
No.  If bit2fasm is given an identical input bitstream, it will generate identical FASM.  We only care if the bitstream is exactly the same.

Reply:
Okay great!